### ISSUE TITLE
Repo policy allows invalid negative values

### ISSUE THREAD

**Issue by mitchplze** [garethgeorge/backrest#435]
```
error: forget: get snapshots for repo xxx: command "/bin/restic-0.17.0 forget --json --keep-hourly -15 --keep-daily 90 -o sftp.args=-oBatchMode=yes --ta..." failed: exit status 1
Output:
invalid argument "-15" for "-H, --keep-hourly" flag: negative values not allowed, use 'unlimited' instead
```

When I scrolled down on the repo settings page, my scroll wheel accidentally scrolled the counter down to a negative value and I didn't notice. This causes `forget`'s to break, as it's a negative number. 

Suggest that this field and probably others, should not allow negative values at all.

<img width="603" alt="image" src="https://github.com/user-attachments/assets/43743f74-a4f2-465c-9d69-db7a50836091">

**Comment by garethgeorge** [comment]
Good point, Backrest should probably be limiting the ranges of these fields.
